theano/sandbox/rng_mrg.py

```diff
@@ -1014,7 +1014,7 @@ def c_code(self, node, nodename, inp, out, sub):
         """ % locals()
 
     def c_code_cache_version(self):
-        return (7, self.GpuKernelBase_version)
+        return (7,)
```

Comment:
This was something that was forgotten when doing the conversion to multi-gpu.  I just noticed it during the rename.